### PR TITLE
[#105181922] Paralelize job deployment

### PR DESCRIPTION
**Pull request by mtekel**
Move consul serial updates false to a place that seems to be more
authoritative (meaning, it is more likely to define the final value in
the manifest).

### CODE REVIEW

**Comment by mtekel** [comment]
Tested in GCE, remains to be tested in AWS.

File: manifests/templates/deployments/cf-jobs.yml [hunk 1]
```diff
@@ -1,12 +0,0 @@
-jobs:
-  - <<: (( merge ))
-  - name: consul_z1
-    <<: (( merge ))
-    update:
-      max_in_flight: 1
```
Comment (actionjack) [inline manifests/templates/deployments/cf-jobs.yml:6]: Should we be removing `max_in_flight` ?

Comment (mtekel) [inline manifests/templates/deployments/cf-jobs.yml:6]: Somehow this time we are not overriding whole `update` hash but just a value of serial. That's spiff.... you never really know for sure what's it going to do.

**Comment by actionjack** [comment]
LGTM :+1:

**Pull request merged**